portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,67 @@
+<head>
+  <title>Jason Han</title>
+
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- Local imports  -->
+  <link rel="stylesheet" type="text/css" href="css/blog.css">
+  <link rel="stylesheet" type="text/css" href="css/global.css">
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script type="text/javascript" src="nav.js"></script>
+
+  <!-- Fonts -->
+  <link href="https://fonts.googleapis.com/css?family=Quicksand|Josefin+Sans:300|Raleway" rel="stylesheet">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe"
+          crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="navbar-wrapper">
+          <span class="navbar-toggle" id="js-navbar-toggle">
+              <i class="fas fa-bars"></i>
```

Comment:
same question here, is this `i` tag doing anything

Reply:
More old code!